### DISCUSSION TITLE
[office-js] Missing member actions

### DISCUSSION THREAD

**Discussion by azuken** [DefinitelyTyped/DefinitelyTyped#58117] [Issues with a @types package] [answered]
Office global variable is missing `actions` member (but `Actions` interface is here). 

Here is the documentation for this member : https://docs.microsoft.com/en-us/javascript/api/office/office.actions?view=common-js-preview#associate

And a usage example : https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/autolaunch#supported-events

It would be nice to have this implemented !

Regards

[a comment by an automated account is not shown]

**Reply by azuken** [reply]
@OfficeDev

**Reply by AlexJerabek** [reply]
Hi @azuken,
Thanks for raising this issue. It looks like that global variable is present in the current office-js-preview type definitions, but not office-js. Let me confirm that this is unintentional and, if appropriate, I'll create a PR to fix the problem.

**Reply by AlexJerabek** [reply]
Here's the PR, under review now: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58119

**Comment by ElizabethSamuel-MSFT** [comment, marked as answer]
@azuken PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58119 has been merged and the member should now be available.
